app/views/mappings/new.html.erb

```diff
@@ -4,8 +4,8 @@
 <%= bootstrap_form_for(:mapping, url: mappings_path) do |f| %>
   <%= f.alert_message 'Please fix the errors below.' %>
 
-  <%= f.select :user, user_list, :include_blank => true, help: user_list_help  %>
-  <%= f.select :app, app_list.map{|uri| [get_app_name(uri), uri]}, :include_blank => true, help: app_list_help %>
+  <%= f.select :user, user_list.map{ |user| [full_username(user), user] }, :include_blank => true, help: user_list_help  %>
```

Comment:
I think the dropdown display value should contain both the gecos and the username. For example, the gecos for an0047 and efranz, both bmishiny users, will be "Eric Franz". Of course, I'm the exception to the rule, and most people will have only 1 account after, but even then it could be useful to verify what username I'm actually sharing with.

```
irb(main):006:0> puts Etc.getgrnam('bmishiny').mem.map { |user| "#{Etc.getpwnam(user).gecos.strip} - #{user}" }
Morgan Rodgers - mrodgers
Hatice Gulcin Ozer - osu1039
Maciej Pietrzak - osu8150
Sundar Gadepalli - osu8903
Eric Franz - efranz
Eric Franz - an0047
BISR Developers - bmiadmin
Eric Franz - osu10424
```